lib/ansible/module_utils/csharp/Ansible.Basic.cs

```diff
@@ -248,6 +250,33 @@ public void Deprecate(string message, string version)
             LogEvent(String.Format("[DEPRECATION WARNING] {0} {1}", message, version));
         }
 
+        public void Deprecate(string message, string version, string collectionName)
```

Comment:
Will need to make `collectionName` optional or else existing code will break

```suggestion
        public void Deprecate(string message, string version, string collectionName = null)
```

Reply:
Same here.